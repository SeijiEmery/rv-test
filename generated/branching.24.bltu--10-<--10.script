load /x 0 generated/branching.24.bltu--10-<--10.hex
writereg 10 -10
writereg 11 -10
writereg 6 1
setpc 0
run 1004
readreg 6