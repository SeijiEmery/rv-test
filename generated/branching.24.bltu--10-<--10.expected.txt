tests/branching.test.s:242
R6 = 0
